sdk/textanalytics/Azure.AI.TextAnalytics/perf/Azure.AI.TextAnalytics.Perf/Infrastructure/TextAnalyticsTest.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Test.Perf;
+
+namespace Azure.AI.TextAnalytics.Perf
+{
+    public abstract class TextAnalyticsTest<TOptions> : PerfTest<TOptions> where TOptions : PerfOptions
```

Comment:
`TOptions` should instead be `CustomCountOptions` if you are keeping that type.

Reply:
**TextAnalyticsTest** is the base perf test
i use **CustomCountOptions** only in the **DetectLanguagePerf** test